support/client/lib/vwf/view/blockly.js

```diff
@@ -232,45 +232,30 @@ define( [ "module", "vwf/view", "jquery", "vwf/model/blockly/JS-Interpreter/acor
                         // If the new node is the same as the old - exit early to prevent 
                         // breaking synchronization.
 
-                        if ( previousActiveNode === newActiveNodeId ) {
+                        if ( previousActiveNode === blocklyNode ) {
                             break;
                         }
 
-                        if ( blocklyNode !== undefined ) {
```

Comment:
Last time I mucked with this code it was to fix a replication issue - did you check to see if this still replicates the visibility across clients? I cant quite recall what the breaking issue was last time.